commands/server.ts

```diff
@@ -42,7 +42,12 @@ export default class ServerCommand extends Command {
       type: <any>'string'
     },
     debug: {
-      description: 'Run in debug mode (add the --debug flag to node, launch node-inspector)',
+      description: 'Run in debug mode (add the --inspect flag to node, launch node-inspector)',
```

Comment:
We should drop the `node-inspector` comment, since that's no longer the actual behavior.